tests/Fixer/Phpdoc/PhpdocAnnotationWithoutDotFixerTest.php

```diff
@@ -137,6 +137,16 @@ public function provideFixCases()
     /**
      * @return bool|null Returns `true` if the class has a single-column ID
      *                   and Returns `false` otherwise.
+     */',
+            ],
+            [
+                '<?php
+    /**
+     * @throws \Exception having whitespaces after dot, yet I am fixed
+     */',
+                '<?php
+    /**
+     * @throws \Exception having whitespaces after dot, yet I am fixed.   '.'
```

Comment:
Please add test case for trailing tab character too, as we are using `\h`

Reply:
ping @kubawerlos

Reply:
Sorry, I've missed that. Done.